### DISCUSSION TITLE
Auth-With_Password Response: ClientResponseError 0: Something went wrong while processing your request

### DISCUSSION THREAD

**Discussion by KarolinaSun2Wheel** [pocketbase/pocketbase#1569] [Q&A]
Hallo,

I keep getting 
ClientResponseError 0: Something went wrong while processing your request

When I am calling
                .collection("users")
                .authWithPassword(email, pass)
with a wright user credentials. I the same time I see in Request logs the status 200, Meta: N/A

While trying to auth a user with wrong credentials the response is: ClientResponseError 400: Failed to authenticate.
Request logs - status: 400, Meta: {
  "errorDetails": "Invalid login credentials.",
  "errorMessage": "Failed to authenticate."
} 

Why do I get: ClientResponseError 0: Something went wrong while processing your request when the user was correctly authenticated?

Regards
Karolina

**Comment by ganigeorgiev** [comment]
You can `try/catch` your request call and inspect the original error, aka. `console.log(err.originalError)`.

`ClientResponseError 0` usually indicates client-side error due to:

- Autocancelled request (you can inspect `err.isAbort` to confirm; see how to change this in  https://github.com/pocketbase/js-sdk#auto-cancellation)

- Failed to establish connection with the PocketBase server for some reason (if you are using node17+ try replacing `localhost` with `127.0.0.1` because of https://github.com/pocketbase/js-sdk/issues/21).

- Other client-side error.

**Reply by KarolinaSun2Wheel** [reply]
Thanks Gani,

you have solved it in [https://github.com/pocketbase/js-sdk/issues/16]

as you said: "native form submit event is not stopped/prevented and it is causing page refresh/redirect while the fetch call is being processed."

so the solution was: e.preventDefault();

Have a nice day!
